poms/foundation/pom.xml

```diff
@@ -212,16 +212,16 @@
         <basepom.maven.version>3.6.0</basepom.maven.version>
 
         <!-- https://github.com/pmd/pmd/releases -->
-        <dep.pmd.version>6.49.0</dep.pmd.version>
+        <dep.pmd.version>6.50.0</dep.pmd.version>
 
         <!-- https://github.com/checkstyle/checkstyle/releases -->
-        <dep.checkstyle.version>10.3.3</dep.checkstyle.version>
+        <dep.checkstyle.version>10.3.4</dep.checkstyle.version>
 
         <!-- https://github.com/spotbugs/spotbugs/releases -->
-        <dep.spotbugs.version>4.7.1</dep.spotbugs.version>
+        <dep.spotbugs.version>4.7.2</dep.spotbugs.version>
 
         <!-- https://github.com/apache/maven-dependency-analyzer/tags -->
-        <dep.dependency-analyzer.version>1.12.0</dep.dependency-analyzer.version>
+        <dep.dependency-analyzer.version>1.13.0</dep.dependency-analyzer.version>
```

Comment:
The dependency analyzer is a steaming mess. I tend to not touch it unless the dependency plugin requires an update.